opennlp-tools/src/test/java/opennlp/tools/formats/leipzig/LeipzigLanguageSampleStreamTest.java

```diff
@@ -34,6 +35,9 @@ public class LeipzigLanguageSampleStreamTest {
   private static String testDataPath = LeipzigLanguageSampleStreamTest.class
       .getClassLoader().getResource("opennlp/tools/formats/leipzig/samples").getPath();
 
+  @TempDir
```

Comment:
Might toogle GH action Windows builds under certain conditions. But looks like IT worked this time.